actgen/change_q.py

```diff
@@ -11,87 +11,48 @@
 import torch
 from tqdm import tqdm
 
-from . import utils
-from . import wrappers as wrap
+from .utils import Trial
 from .agents import DQNAgent, DirectedQNet, ActionDQNAgent
 from .gscore import plot_confusion_matrix, calc_g_score, build_confusion_matrix
 
 logging.basicConfig(level=logging.INFO)
 
 
-class ManipulationTrial:
+class ManipulationTrial(Trial):
     def __init__(self, test=True):
         args = self.parse_args()
         self.params = self.load_hyperparams(args)
         self.params['regularization'] = 'None'
         if self.params['test'] or test:
             self.params['test'] = True
         self.setup()
-
+    
     def parse_args(self):
-        parser = argparse.ArgumentParser(formatter_class=argparse.ArgumentDefaultsHelpFormatter)
-        # yapf: disable
-        parser.add_argument('--env_name', type=str, default='CartPole-v0',
-                            choices=['CartPole-v0', 'Pendulum-v0', 'LunarLander-v2'],
-                            help='Which gym environment to use')
-        parser.add_argument('--agent', type=str, default='dqn',
-                            choices=['dqn', 'random', 'action_dqn'],
-                            help='Which agent to use')
-        parser.add_argument('--duplicate', '-d', type=int, default=5,
-                            help='Number of times to duplicate actions')
-        parser.add_argument('--seed', '-s', type=int, default=0,
-                            help='Random seed')
-        parser.add_argument('--hyperparams', type=str, default='hyperparams/defaults.csv',
-                            help='Path to hyperparameters csv file')
-        parser.add_argument('--test', default=False, action='store_true',
-                            help='Enable test mode for quickly checking configuration works')
-        parser.add_argument('--load', type=str, default='results/default_exp/dqn_seed0_none_best.pytorch',
+        manipulation_parser = argparse.ArgumentParser(
+            formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+            parents=[self.parse_common_args()]
+        )
+        # manipulation args
+        manipulation_parser.add_argument('--load', type=str, default='results/default_exp/dqn_seed0_none_best.pytorch',
                             help='Path to a saved model that"s fully trained')
```

Comment:
For strings with `'` in them, I prefer to use `" "` and avoid needing escape characters:

```python
help="Path to a saved model that's fully trained")
```